Test/dafny0/ControlStructures.dfy.expect

```diff
@@ -1,17 +1,17 @@
-ControlStructures.dfy(8,2): Error: missing case in case statement: Purple
+ControlStructures.dfy(8,8): Error: missing case in case statement: Purple
```

Comment:
The new error location is okay, but I think the old one is slightly better. Is it only a matter of using a different `.tok` field in the implementation (if so, please change it) or does this involve some token that's passed in through several methods in the implementation?